src/widgets/bookmark/BookmarkWidgetTile.tsx

```diff
@@ -157,6 +157,11 @@ const definition = defineWidget({
       ],
       defaultValue: 'autoGrid',
     },
+    isOpeningNewTab: {
```

Comment:
Please set this option per link, not globally on the widget, so certain links can be opened in a new tab.
Check the existing code, how this is done

Reply:
could you give me the reference

Reply:
Sure:

https://github.com/ajnart/homarr/blob/4ce300d024f1df75690630a24177281571aeb04d/src/widgets/bookmark/BookmarkWidgetTile.tsx#L59C12-L140

Reply:
Updated!. Now selected links can be opened in a new tab.